plugins/tech-insights/api-report.md

```diff
@@ -5,12 +5,22 @@
 ```ts
 /// <reference types="react" />
 
+import { ApiRef } from '@backstage/core-plugin-api';
 import { BackstagePlugin } from '@backstage/core-plugin-api';
+import { CheckResult } from '@backstage/plugin-tech-insights-common';
+import { EntityName } from '@backstage/catalog-model';
+import { default as React_2 } from 'react';
 import { RouteRef } from '@backstage/core-plugin-api';
 
 // @public (undocumented)
 export const EntityTechInsightsScorecardContent: () => JSX.Element;
 
+// Warning: (ae-forgotten-export) The symbol "TechInsightsApi" needs to be exported by the entry point index.d.ts
+// Warning: (ae-missing-release-tag) "techInsightsApiRef" is exported by the package, but it is missing a release tag (@alpha, @beta, @public, or @internal)
```

Comment:
Let's make sure we're not adding new warnings to the API report, which would involve TsDoc comments with a `@public` tag where needed, and making sure types are exported as needed.